src/app/game-state/achievement.service.ts

```diff
@@ -454,11 +454,11 @@ export class AchievementService {
       unlocked: false
     },
     {
-      name: "Unlimited Taels",
-      description: "Your family has unlocked the secrets of compound interest. You probably never have to worry about money again.",
-      hint: "Family first. Especially in matters of money.",
+      name: "Not Unlimited Taels",
```

Comment:
Is there a reason for changing this rather than just adding a new achievement? I'd rather just do both.

Reply:
I don't see bloodline 4 as being any notable milestone that warrants being an achievement. There's no unlock tied to it, and it's not like the fact that it's infinite money is something particularly hard to figure out.